### ISSUE TITLE
Upgrade to V1.14.0 is a bad dream

### ISSUE THREAD

**Issue by weinand** [Microsoft/vscode-react-native#471]
_From @adyanced on July 12, 2017 8:33_

I am using VS Code to debug a RN project. as I upgrade to V1.14.0 this morning, a bad dream is coming:
1. The 'Stop Debug' button act as 'Reset', have to double-click to stop debugging
2. The 'Reset' button reload the apk endlessly
3. The apk always stopped at 
![image](https://user-images.githubusercontent.com/6321512/28108896-b4f47ce2-671f-11e7-96cb-61045d81e284.png)
except the first load

How to downgrade VS Code?

_Copied from original issue: Microsoft/vscode#30480_

**Comment by weinand** [comment]
How does your launch configuration look like?

You can always downgrade by installing the previous version from the VS Code website:
https://code.visualstudio.com/updates/v1_13

You might want do disable auto-update by adding this to your settings: "update.channel": "none",

**Comment by weinand** [comment]
Or you are suffering from this:
from the VS Code 1.14 [release notes](https://code.visualstudio.com/updates/v1_14#_nodejs-v80-related-changes):
> In this release, Node.js debugging will assume the "inspector" protocol whenever the runtimeExecutable attribute of a launch configuration is set. If you see your debug configurations failing, this change could be the reason. To fix this, you can either upgrade Node.js to version 8.x or you can explicitly force the use of the "legacy" protocol by adding "protocol": "legacy" to your launch configuration.

**Comment by weinand** [comment]
_From @adyanced on July 12, 2017 8:45_

This is launch.json:

```js
{
    "version": "0.2.0",
    "configurations": [
{
    "name": "Debug Android",
    "program": "${workspaceRoot}/.vscode/launchReactNative.js",
    "type": "reactnative",
    "request": "launch",
    "platform": "android",
    "sourceMaps": true,
    "outDir": "${workspaceRoot}/.vscode/.react"
},
        {
            "name": "Debug iOS",
            "program": "${workspaceRoot}/.vscode/launchReactNative.js",
            "type": "reactnative",
            "request": "launch",
            "platform": "ios",
            "target": "iPhone 5s",
            "sourceMaps": true,
            "outDir": "${workspaceRoot}/.vscode/.react"
        },
        {
            "name": "Attach to packager",
            "program": "${workspaceRoot}/.vscode/launchReactNative.js",
            "type": "reactnative",
            "request": "attach",
            "sourceMaps": true,
            "outDir": "${workspaceRoot}/.vscode/.react"
        },
        {
            "name": "Debug in Exponent",
            "program": "${workspaceRoot}/.vscode/launchReactNative.js",
            "type": "reactnative",
            "request": "launch",
            "platform": "exponent",
            "sourceMaps": true,
            "outDir": "${workspaceRoot}/.vscode/.react"
        }
    ]
}
```

**Comment by weinand** [comment]
You are using the React Native extension (which is not a standard part of VS Code).
I'll move this issue to the React Native repository.

**Comment by iainroyle** [comment]
Confirmed. Unable to launch app in this release (except for the first time!) Reload from the app causes the entire app to be re-deployed again, which then fails with a network error every time.

**Comment by RoJoHub** [comment]
So am i

**Comment by vladimir-kotikov** [comment]
Resolved in #472
